Client/Frontend/Browser/Tabs/InactiveTabItemCell.swift

```diff
@@ -0,0 +1,142 @@
+        separatorInset = UIEdgeInsets(top: 0,
+                                      left: InactiveTabItemCellModel.UX.ImageSize + 2 *
+                                      InactiveTabItemCellModel.UX.BorderViewMargin,
+                                      bottom: 0,
+                                      right: 0)
+        backgroundColor = .clear
+    }
+
+    let containerView = UIView()
+    let midView = UIView()
+
+    private var containerViewLeadingConstraint: NSLayoutConstraint!
+
+    func initialViewSetup() {
+        self.selectionStyle = .default
+        midView.addSubview(titleLabel)
+        containerView.addSubviews(bottomSeparatorView)
+        containerView.addSubview(leftImageView)
+        containerView.addSubview(midView)
+
+        contentView.addSubview(containerView)
+        bringSubviewToFront(containerView)
+
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        bottomSeparatorView.translatesAutoresizingMaskIntoConstraints = false
+        leftImageView.translatesAutoresizingMaskIntoConstraints = false
+        midView.translatesAutoresizingMaskIntoConstraints = false
+        containerView.translatesAutoresizingMaskIntoConstraints = false
+
+        containerViewLeadingConstraint = containerView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor)
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            containerViewLeadingConstraint,
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+
+            leftImageView.heightAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.widthAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                   constant: InactiveTabItemCellModel.UX.ImageViewLeadingConstant),
+            leftImageView.centerYAnchor.constraint(equalTo: midView.centerYAnchor),
+            leftImageView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor,
+                                               constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+            leftImageView.bottomAnchor.constraint(lessThanOrEqualTo: containerView.bottomAnchor,
+                                                  constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+
+            midView.topAnchor.constraint(equalTo: containerView.topAnchor),
+            midView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            midView.leadingAnchor.constraint(equalTo: leftImageView.trailingAnchor, constant: 13),
+            midView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor, constant: -7),
```

Comment:
I think designs wants us to use spacing with 4 increments. I.e. 8, 12, 16, etc. Maybe we can confirm what should the spacing be here with a designer? We could also put this in the in the UX struct

Reply:
Will check with designers.